help/saml-authentication.md

```diff
@@ -177,6 +178,33 @@ providers.
 
     Your Keycloak realm URL will look something like this: `https://keycloak.example.com/auth/realms/yourrealm`.
 
+{tab|auth0}
+
+{!upgrade-to-plus-if-needed.md!}
+
+1. Set up SAML authentication by following [Auth0's documentation](https://auth0.com/docs/authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider#configure-saml-sso-in-auth0)
+   to create a new application. You don't need to save the certificates or other information detailed.
+   All we will need is the **SAML Metadata URL**.
+1. In the **Addon: SAML2 Web App** **Settings** tab set the **Application Callback URL** to
+   `https://auth.zulipchat.com/complete/saml/`.
+1. Edit the **Settings** section to match:
+```
+{
+  "audience": "https://zulipchat.com",
+  "mappings": {
+    "email": "email",
+    "given_name": "first_name",
+    "family_name": "last_name"
+  },
+  "binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
+}
+```
```

Comment:
To explain for review purposes, this is in a code block, because in Auth0 config you actually configure this stuff like a code block too:
![image](https://user-images.githubusercontent.com/45007152/228684862-813d3393-23f7-4cc7-a66b-e90fd1f757a7.png)